### PR TITLE
FISH-7168 SNMP4J Link

### PR DESCRIPTION
**Pull request by JamesHillyard**
Update SNMP4J link to the payara compatible repackaged jar.

### CODE REVIEW

File: community/docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Extensions/Notifiers/SNMP.adoc [hunk 1]
```diff
@@ -8,7 +8,9 @@ The **S**imple **N**etwork **M**anagement **P**rotocol is a standard Internet Pr
 
 - link:https://nexus.payara.fish/repository/payara-artifacts/fish/payara/extensions/notifiers/snmp-notifier-core/2.0/snmp-notifier-core-2.0.jar[SNMP Notifier Core 2.0]
 - link:https://nexus.payara.fish/repository/payara-artifacts/fish/payara/extensions/notifiers/snmp-notifier-console-plugin/2.0/snmp-notifier-console-plugin-2.0.jar[SNMP Notifier Console Plugin 2.0]
-- link:https://nexus.payara.fish/repository/payara-artifacts/org/snmp4j/snmp4j/2.5.3-payara.p1/snmp4j-2.5.3-payara.p1.jar[Dependency: SNMP4J 2.5.3-payara.p1]
+- link:https://nexus.payara.fish/repository/payara-artifacts/org/snmp4j/snmp4j/3.7.5/snmp4j-3.7.5.jar[Dependency: SNMP4J 3.7.5]
```
Comment (Pandrex247) [inline community/docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Extensions/Notifiers/SNMP.adoc:11]: I dislike republishing things as the same version when they aren't the same.
It should either be published as 3.7.5.payara-p1, or simply published at the original repackaged coordinates.

File: enterprise/docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Logging and Monitoring/Notification Service/Notifiers/SNMP Notifier.adoc [hunk 2]
```diff
@@ -13,7 +13,9 @@ to cover IoT devices and specialized equipment connected to the cloud.
 
 - link:https://nexus.payara.fish/repository/payara-enterprise-downloadable-artifacts/fish/payara/extensions/notifiers/snmp-notifier-core/2.0-enterprise/snmp-notifier-core-2.0-enterprise.jar[SNMP Notifier Core 2.0-enterprise]
 - link:https://nexus.payara.fish/repository/payara-enterprise-downloadable-artifacts/fish/payara/extensions/notifiers/snmp-notifier-console-plugin/2.0-enterprise/snmp-notifier-console-plugin-2.0-enterprise.jar[SNMP Notifier Console Plugin 2.0-enterprise]
-- link:https://repo1.maven.org/maven2/org/snmp4j/snmp4j/2.5.3-payara.p1/snmp4j-2.5.3-payara.p1.jar[Dependency: SNMP4J 2.5.3-payara.p1]
+- link:https://nexus.payara.fish/repository/payara-artifacts/org/snmp4j/snmp4j/3.7.5/snmp4j-3.7.5.jar[Dependency: SNMP4J 3.7.5]
```
Comment (Pandrex247) [inline enterprise/docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Logging and Monitoring/Notification Service/Notifiers/SNMP Notifier.adoc:16]: I dislike republishing things as the same version when they aren't the same.
It should either be published as 3.7.5.payara-p1, or simply published at the original repackaged coordinates.

**Review by Pandrex247 — CHANGES_REQUESTED** [review]

File: community/docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Extensions/Notifiers/SNMP.adoc (hunk 1 above)
Comment (JamesHillyard) [reply community/docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Extensions/Notifiers/SNMP.adoc]: Republished at the original repackaged coordinates, deleted the old upload from Nexus and the link has been updated.

File: enterprise/docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Logging and Monitoring/Notification Service/Notifiers/SNMP Notifier.adoc (hunk 2 above)
Comment (JamesHillyard) [reply enterprise/docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Logging and Monitoring/Notification Service/Notifiers/SNMP Notifier.adoc]: Republished at the original repackaged coordinates, deleted the old upload from Nexus and the link has been updated.

**Review by Pandrex247 — APPROVED** [review]

**Pull request merged**